src/fitnesse/slim/SlimExecutionContext.java

```diff
@@ -53,7 +53,7 @@ public void create(String instanceName, String className, Object[] args)
 
     public void addPath(String path) {
         if (!paths.contains(path)) {
-            paths.add(path);
+            paths.add(0, path);
```

Comment:
Previously, we append the path at the end of the list. However, each time we looked through the paths imported to find a class, the list was reversed. I removed the piece of code with was a little CPU consuming for nothing if we simply add at the head of the list.